kusto/ingest/file_options.go

```diff
@@ -219,9 +219,9 @@ const (
 	SingleJSON DataFormat = properties.SingleJSON
 )
 
-// InferFormatFromFileName  looks at the file name and tries to discern what the file format is
+// InferFormatFromFileName looks at the file name and tries to discern what the file format is
 func InferFormatFromFileName(fName string) DataFormat {
-	return properties.DataFormatDiscovery(fName)
+	return properties.DataFormatDiscovery(properties.RemoveQueryParamsFromUrl(fName))
```

Comment:
How come this is relevant here?

Reply:
Every place I did this, it's a place where:
1) Secrets are unnecessary for the called function to work.
2) The function (perhaps transitively) can throw an error when doing something with the URL. I'm less familiar with Go's error handling, but my concern is that the exception may include the parameter.